### DISCUSSION TITLE
Can't load "cloaking" module

### DISCUSSION THREAD

**Discussion by youshitsune** [inspircd/inspircd#2040] [Support]
**Description**
When I try to load cloaking module, it doesn't load and systemctl status inspircd says Invalid Argument. But, all other modules load without problem.

**Steps to reproduce the issue:**

1. Add <module name="cloaking"> to inspircd.conf
2. Save file
3. Restart inspircd.service

**Describe the results you received:**
Not starting inspircd daemon.

**Describe the results you expected:**
Starting inspircd daemon

**Output of `./bin/inspircd --version`:**
InspIRCd-3.8.1

**Comment by sadiepowell** [comment]
I'd guess that you didn't specify an adequate `<cloak>` tag or you are missing the md5 module. Check your logs to see what is actually wrong. That said, 3.8.1 is ancient so there may be some bug which has been fixed since then.

https://docs.inspircd.org/3/modules/cloaking/#cloak

**Reply by youshitsune** [reply]
I used a full cloak and loaded md5 module.

**Reply by sadiepowell** [reply]
You need to check the [InspIRCd logs](https://docs.inspircd.org/3/configuration/#log) or systemd journal to see what it is failing on. I'd guess your cloak key isn't long enough though.

**Reply by youshitsune** [reply]
My cloak key is 128 characters long.

**Reply by sadiepowell** [reply]
Did you manage to check your log to see what it was failing on?

**Reply by youshitsune** [reply]
Sorry, I've removed completely everything. I don't use it anymore. I could install it again and try to set it up, if you want?
